ee/clickhouse/queries/funnels/funnel_trends.py

```diff
@@ -27,8 +27,9 @@ class ClickhouseFunnelTrends(ClickhouseFunnelNew):
     In the default case, {from_step} is the first step of the funnel, meaning entrance from the very beginning.
 
     Now, our time series is based on a sequence of {entrance_period}s, each starting at {entrance_period_start}
-    and ending _right before the next_ {entrance_period_start}, and a person having entered the funnel
-    is only counted towards a single {entrance_period}'s numbers (hich is reflected in the time series).
+    and ending _right before the next_ {entrance_period_start}, and a person can only be counted once
+    towards a single {entrance_period}'s numbers (which is reflected in the time series). But a person may be a part of
+    multiple entrance periods. (i.e. the person performs the {from_step} in separate {entrance_period}s)
```

Comment:
@Twixes I like this description of funnel trends because it lays the function out very clearly. One detail I just remembered is that a person can be counted more than once if they're in different entrance periods

Reply:
Definitely clearer, though actually I reworked it a lot more, because it was not _entirely_ factual (the part about `from_step` and `to_step` specifically). Now it's 100% accurate, please take a look if anything's missing!